examples/pxScene2d/src/CMakeLists.txt

```diff
@@ -288,6 +288,24 @@ elseif (WIN32)
     set(PXCORE_LIB_LOCATION ${PXCOREDIR}/build/win/Release)
     set(PXSCENE_LINK_DIRECTORIES ${PXSCENE_LINK_DIRECTORIES} ${PXCORE_LIB_LOCATION})
     set(PLATFORM_SOURCES win/pxContextUtils.cpp)
+	if ((SUPPORT_NODE) AND (NOT USE_NODE_8))
```

Comment:
These lines had been removed as redundant. Are they really needed for node 6?

Reply:
these files are not compiled and facing build errors for windows